apps/src/templates/manageStudents/ConfirmRemoveStudentDialog.story.jsx

```diff
@@ -3,48 +3,31 @@ import ConfirmRemoveStudentDialog, {
   MINIMUM_TEST_PROPS
 } from './ConfirmRemoveStudentDialog';
 
-const STORY_PROPS = {
-  ...MINIMUM_TEST_PROPS,
-  hideBackdrop: true
+export default {
+  title: 'ManageStudents/ConfirmRemoveStudentDialog',
```

Comment:
my understanding what that we weren't using subcategories like this, so this should just be `ConfirmRemoveStudentDialog`. But it's very possible that understanding is outdated :)

Reply:
I added this subcategories based on [other storybooks](https://github.com/code-dot-org/code-dot-org/blob/6c6baab81006dc2adc73986924c1149dcf2c0373/apps/src/templates/certificates/StudentsBeyondHoc.story.js). It seem reasonable and understandable to me, but I can change this.

Reply:
Nah don't change it, especially if we're using it other places